### DISCUSSION TITLE
lc_grid and lc_class

### DISCUSSION THREAD

**Discussion by langford77840** [UMEP-dev/UMEP#370] [Python]
Could I know what lc_grid and lc_class in "def Tgamps_v1" in Tgmaps_v1.py are?

I'd like to use this function but not clear what these variables are.
It'd be really helpful if you just give me a brief example of these variables.

Thanks in advance for your consideration.

**Comment by biglimp** [comment]
**lc_grid** is the ground cover raster used in UMEP consisting of five classes. 
The **lc_class** is parameters for each ground cover taken from **landcoverclasses_2016a.txt**. 
See the SOLWEIG-manual for more info.
